### PR TITLE
Update zh-t.xml

### PR DESCRIPTION
**Pull request by Kaitul**
Update and improve the translation

### CODE REVIEW

File: lang/zh-t.xml [hunk 1]
```diff
@@ -271,33 +271,33 @@
...
+				<details>詳細資料</details>
+				<updated>更新日期</updated>
 			</info>
 			<load name="載入">
 				<cancel_loading>確定要取消載入嗎？</cancel_loading>
 				<no_saved_data>找不到保存資料。</no_saved_data>
-				<load_complete>已載入。</load_complete>
-				<loading>載入中……</loading>
+				<load_complete>載入完成。</load_complete>
+				<loading>載入中…</loading>
```
Comment (Croden1999) [inline lang/zh-t.xml:300]: do not change this, is based on psvita system lang.

File: lang/zh-t.xml [hunk 2]
```diff
@@ -306,47 +306,47 @@
 記憶卡的可用容量不足。</could_not_save>
 				<not_free_space>記憶卡的可用容量不足。</not_free_space>
 				<new_saved_data>新的保存資料</new_saved_data>
-				<saving_complete>已保存。</saving_complete>
+				<saving_complete>保存完成。</saving_complete>
```
Comment (Croden1999) [inline lang/zh-t.xml:309]: here

File: lang/zh-t.xml [hunk 3]
```diff
@@ -306,47 +306,47 @@
...
 				<save_the_data>確定要保存資料嗎？</save_the_data>
-				<saving>保存中……</saving>
-				<warning_saving>保存中……
+				<saving>保存中…</saving>
+				<warning_saving>保存中…
```
Comment (Croden1999) [inline lang/zh-t.xml:312]: and here.

File: lang/zh-t.xml [hunk 4]
```diff
@@ -306,47 +306,47 @@
...
 				<overwrite_saved_data>確定要覆寫此保存資料嗎？</overwrite_saved_data>
 			</save>
 		</save_data>
 	</dialog>
 
 	<game_data>
-		<app_close>結束下列應用程式。</app_close>
-		<data_delete>刪除的資料：</data_delete>
+		<app_close>下列應用程式將會關閉。</app_close>
+		<data_delete>將要刪除的資料：</data_delete>
```
Comment (Croden1999) [inline lang/zh-t.xml:321]: also this.

**Review by Croden1999 — CHANGES_REQUESTED** [review]

File: lang/zh-t.xml [hunk 5]
```diff
@@ -2,75 +2,75 @@
...
-		<shader_cache>着色器緩存</shader_cache>
-		<shader_log>着色器日志</shader_log>
+		<open_folder>開啟資料夾</open_folder>
+		<addcont>追加內容</addcont>
+		<license>授權</license>
+		<shader_cache>著色器快取</shader_cache>
+		<shader_log>著色器記錄</shader_log>
 		<update_history>更新履歷</update_history>
-		<history_version>{} 版本</history_version>
+		<history_version>{}版本</history_version>
```
Comment (Croden1999) [inline lang/zh-t.xml:57]: also is based psvita lang

File: lang/zh-t.xml [hunk 6]
```diff
@@ -2,75 +2,75 @@
...
+取決於它的大小和您的硬體。</app_delete_note>
+		<save_delete>確定要刪除此保存資料嗎？</save_delete>
 		<eligible>有權</eligible>
 		<ineligible>無權</ineligible>
 		<level>等級</level>
 		<name>名稱</name>
 		<trophy_earning>獲得獎盃</trophy_earning>
 		<parental_controls>家長監控</parental_controls>
-		<updated>更新日</updated>
+		<updated>更新日期</updated>
```
Comment (Croden1999) [inline lang/zh-t.xml:69]: here

File: lang/zh-t.xml [hunk 7]
```diff
@@ -2,75 +2,75 @@
...
-		<history_version>{} 版本</history_version>
+		<history_version>{}版本</history_version>
 		<information>資訊</information>
-		<app_delete>刪除此應用程式及所有相關資料（包括保存資料）。</app_delete>
+		<app_delete>將會刪除此應用程式及所有相關資料（包括保存資料）。</app_delete>
 		<app_delete_note>刪除應用程式可能需要一段時間
-取決於它的大小和您的硬件。</app_delete_note>
-		<save_delete>確定要刪除此保存資料嗎?</save_delete>
+取決於它的大小和您的硬體。</app_delete_note>
+		<save_delete>確定要刪除此保存資料嗎？</save_delete>
```
Comment (Croden1999) [inline lang/zh-t.xml:62]: here

File: lang/zh-t.xml [hunk 8]
```diff
@@ -2,75 +2,75 @@
...
+		<addcont>追加內容</addcont>
+		<license>授權</license>
+		<shader_cache>著色器快取</shader_cache>
+		<shader_log>著色器記錄</shader_log>
 		<update_history>更新履歷</update_history>
-		<history_version>{} 版本</history_version>
+		<history_version>{}版本</history_version>
 		<information>資訊</information>
-		<app_delete>刪除此應用程式及所有相關資料（包括保存資料）。</app_delete>
+		<app_delete>將會刪除此應用程式及所有相關資料（包括保存資料）。</app_delete>
```
Comment (Croden1999) [inline lang/zh-t.xml:59]: here

File: lang/zh-t.xml [hunk 9]
```diff
@@ -83,7 +83,7 @@
 		<no>否</no>
 		<select_all>全選</select_all>
 		<select>選擇</select>
-		<please_wait>請稍候……</please_wait>
+		<please_wait>請稍候…</please_wait>
```
Comment (Croden1999) [inline lang/zh-t.xml:86]: here

File: lang/zh-t.xml [hunk 10]
```diff
@@ -231,20 +231,20 @@
 		</saved_data>
 		<theme>主題</theme>
 		<free_space>可用容量</free_space>
-		<clear_all>全解除</clear_all>
+		<clear_all>全部清除</clear_all>
```
Comment (Croden1999) [inline lang/zh-t.xml:234]: and here

File: lang/zh-t.xml [hunk 11]
```diff
@@ -271,33 +271,33 @@
...
+		<toggle_gui_visibility>切換圖形化使用者介面可見性</toggle_gui_visibility>
 		<error>錯誤</error>
-		<error_duplicate_key>該按鈕已被使用並綁定其他按鈕或被預留。</error_duplicate_key>
+		<error_duplicate_key>此按鈕已繫結至其他按鈕或已被保留。</error_duplicate_key>
 	</controls>
 
 	<dialog>
 		<trophy>
-			<preparing_start_app>正在準備啟動應用程式……</preparing_start_app>
+			<preparing_start_app>正在準備啟動應用程式…</preparing_start_app>
```
Comment (Croden1999) [inline lang/zh-t.xml:284]: there is right, not need change, is based on psvita lang.

**Review by Croden1999 — CHANGES_REQUESTED** [review]

File: lang/zh-t.xml [hunk 12]
```diff
@@ -271,33 +271,33 @@
...
-				<deletion_complete>已刪除。</deletion_complete>
-				<delete_saved_data>確定要刪除此保存資料嗎?</delete_saved_data>
+				<deletion_complete>刪除完成。</deletion_complete>
+				<delete_saved_data>確定要刪除此保存資料嗎？</delete_saved_data>
 			</delete>
 			<info>
-				<details>詳細內容</details>
-				<updated>更新日</updated>
+				<details>詳細資料</details>
+				<updated>更新日期</updated>
```
Comment (Croden1999) [inline lang/zh-t.xml:294]: here

File: lang/zh-t.xml [hunk 13]
```diff
@@ -271,33 +271,33 @@
...
-			<preparing_start_app>正在準備啟動應用程式……</preparing_start_app>
+			<preparing_start_app>正在準備啟動應用程式…</preparing_start_app>
 		</trophy>
 		<save_data>
 			<delete>
 				<cancel_deleting>確定要取消刪除嗎？</cancel_deleting>
-				<deletion_complete>已刪除。</deletion_complete>
-				<delete_saved_data>確定要刪除此保存資料嗎?</delete_saved_data>
+				<deletion_complete>刪除完成。</deletion_complete>
+				<delete_saved_data>確定要刪除此保存資料嗎？</delete_saved_data>
```
Comment (Croden1999) [inline lang/zh-t.xml:290]: here

File: lang/zh-t.xml [hunk 14]
```diff
@@ -306,47 +306,47 @@
...
 		<cat>類別</cat>
 		<last_time>最近使用時間</last_time>
 		<tit>標題</tit>
-		<refresh>刷新</refresh>
+		<refresh>更新</refresh>
 	</home_screen>
 
 	<indicator>
-		<app_added_home>已追加應用程式至主頁畫面。</app_added_home>
+		<app_added_home>應用程式已追加至主頁畫面。</app_added_home>
```
Comment (Croden1999) [inline lang/zh-t.xml:344]: here

File: lang/zh-t.xml [hunk 15]
```diff
@@ -306,47 +306,47 @@
...
 		<delete_all>全部刪除</delete_all>
 		<notif_deleted>刪除最新資訊。</notif_deleted>
 		<install_failed>無法安裝。</install_failed>
-		<install_complete>已安裝。</install_complete>
-		<installing>安裝中……</installing>
+		<install_complete>安裝完成。</install_complete>
+		<installing>安裝中…</installing>
```
Comment (Croden1999) [inline lang/zh-t.xml:349]: here

File: lang/zh-t.xml [hunk 16]
```diff
@@ -355,123 +355,123 @@
 		<back>返回</back>
 		<completed_setup>已完成初期設定。
 請盡情享受Vita3K。</completed_setup>
-		<select_language>Select a language.</select_language>
-		<install_firmware>安裝固件。</install_firmware>
-		<install_highly_recommended>Installing both firmware files is highly recommended.</install_highly_recommended>
+		<select_language>選擇語言。</select_language>
```
Comment (Croden1999) [inline lang/zh-t.xml:358]: no need add translate, if no install firmware, there are no display normal text.

File: lang/zh-t.xml [hunk 17]
```diff
@@ -355,123 +355,123 @@
...
+		<install_firmware>安裝韌體。</install_firmware>
+		<install_highly_recommended>強烈建議您安裝這兩個韌體檔案。</install_highly_recommended>
```
Comment (Croden1999) [inline lang/zh-t.xml:360]: also here no need add translate, if no install firmware, there are no display normal text.

File: lang/zh-t.xml [hunk 18]
```diff
@@ -355,123 +355,123 @@
...
 		<installed>已安裝：</installed>
-		<download_firmware>下載固件</download_firmware>
-		<download_font_package>下載字庫包</download_font_package>
-		<install_firmware_file>安裝固件文件</install_firmware_file>
+		<download_firmware>下載韌體</download_firmware>
+		<download_font_package>下載字型套件</download_font_package>
+		<install_firmware_file>安裝韌體檔案</install_firmware_file>
 		<completed>已完成。</completed>
-		<next>Next</next>
+		<next>下一步</next>
```
Comment (Croden1999) [inline lang/zh-t.xml:366]: no need translate

File: lang/zh-t.xml [hunk 19]
```diff
@@ -355,123 +355,123 @@
...
+		<fps>FPS</fps>
 		<avg>平均</avg>
 		<min>最低</min>
 		<max>最高</max>
 	</performance_overlay>
 
 	<settings name="設定">
 		<theme_background name="主題&背景">
-			<default>原音</default>
+			<default>預設</default>
```
Comment (Croden1999) [inline lang/zh-t.xml:454]: do not change this, is based on psvita

File: lang/zh-t.xml [hunk 20]
```diff
@@ -355,123 +355,123 @@
...
 			<theme name="主題">
 				<search>搜尋</search>
 				<find_in_altervista>在Altervista尋找</find_in_altervista>
 				<information name="資訊">
 					<name>名稱</name>
 					<provider>提供者</provider>
-					<updated>更新日</updated>
+					<updated>更新日期</updated>
```
Comment (Croden1999) [inline lang/zh-t.xml:461]: also this

File: lang/zh-t.xml [hunk 21]
```diff
@@ -355,123 +355,123 @@
...
 					<size>檔案大小</size>
 					<version>版本</version>
 					<content_id>內容ID</content_id>
 				</information>
-				<delete>刪除此主題。</delete>
+				<delete>這個主題將被刪除。</delete>
```
Comment (Croden1999) [inline lang/zh-t.xml:466]: here

File: lang/zh-t.xml [hunk 22]
```diff
@@ -355,123 +355,123 @@
...
 			</theme>
 			<start_screen name="開始畫面">
 				<image>圖像</image>
-				<add_image>添加圖像</add_image>
+				<add_image>追加圖像</add_image>
```
Comment (Croden1999) [inline lang/zh-t.xml:470]: here

File: lang/zh-t.xml [hunk 23]
```diff
@@ -355,123 +355,123 @@
...
 			</start_screen>
-			<home_screen_backgrounds name="主頁畫面的背景">
+			<home_screen_backgrounds name="主頁畫面背景">
```
Comment (Croden1999) [inline lang/zh-t.xml:472]: here

File: lang/zh-t.xml [hunk 24]
```diff
@@ -355,123 +355,123 @@
...
 				<delete_background>刪除背景</delete_background>
-				<add_background>添加背景</add_background>
+				<add_background>追加背景</add_background>
```
Comment (Croden1999) [inline lang/zh-t.xml:474]: here

File: lang/zh-t.xml [hunk 25]
```diff
@@ -492,16 +492,16 @@
 					<ime_languages>
 						<lang id="1">丹麥文</lang>
 						<lang id="2">德文</lang>
-						<lang id="262144">英文(英國)</lang>
-						<lang id="4">英文(美國)</lang>
+						<lang id="262144">英文（英國）</lang>
+						<lang id="4">英文（美國）</lang>
```
Comment (Croden1999) [inline lang/zh-t.xml:496]: also here

File: lang/zh-t.xml [hunk 26]
```diff
@@ -492,16 +492,16 @@
...
 						<lang id="8">西班牙文</lang>
 						<lang id="16">法文</lang>
 						<lang id="32">義大利文</lang>
 						<lang id="64">荷蘭文</lang>
 						<lang id="128">挪威文</lang>
 						<lang id="256">波蘭文</lang>
-						<lang id="131072">葡萄牙文(巴西)</lang>
-						<lang id="512">葡萄牙文(葡萄牙)</lang>
+						<lang id="131072">葡萄牙文（巴西）</lang>
+						<lang id="512">葡萄牙文（葡萄牙）</lang>
```
Comment (Croden1999) [inline lang/zh-t.xml:504]: here

File: lang/zh-t.xml [hunk 27]
```diff
@@ -515,19 +515,19 @@
 	<trophy_collection>
 		<search>搜尋</search>
 		<delete_trophy>刪除獎盃</delete_trophy>
-		<trophy_deleted>刪除該使用者已保存的獎杯資訊。</trophy_deleted>
-		<details>詳細內容</details>
-		<earned>獲得日</earned>
+		<trophy_deleted>這位使用者已保存的獎杯資訊將被刪除。</trophy_deleted>
+		<details>詳細資料</details>
+		<earned>獲得日期</earned>
```
Comment (Croden1999) [inline lang/zh-t.xml:520]: is based on psvita lang, revert it.

File: lang/zh-t.xml [hunk 28]
```diff
@@ -515,19 +515,19 @@
...
 		<name>名稱</name>
 		<no_trophies>找不到獎盃。
 遊玩支援獎盃機能的應用程式即可獲得獎盃。</no_trophies>
 		<not_earned>未獲得</not_earned>
 		<original>原始</original>
 		<sort>排列</sort>
 		<trophies>獎盃總數</trophies>
-		<grade>Grade（階級）</grade>
+		<grade>階級</grade>
```
Comment (Croden1999) [inline lang/zh-t.xml:528]: also here

File: lang/zh-t.xml [hunk 29]
```diff
@@ -515,19 +515,19 @@
...
 		<progress>達成率</progress>
-		<updated>更新日</updated>
+		<updated>更新日期</updated>
```
Comment (Croden1999) [inline lang/zh-t.xml:530]: here

**Review by Croden1999 — CHANGES_REQUESTED** [review]

File: lang/zh-t.xml [hunk 30]
```diff
@@ -538,11 +538,11 @@
 		<user_name_used>此名稱已被使用。</user_name_used>
 		<delete_user>刪除使用者</delete_user>
 		<user_delete>請選擇要刪除的使用者。</user_delete>
-		<user_delete_msg>將刪除以下使用者。</user_delete_msg>
-		<user_delete_message>若刪除該用戶，該使用者的保存資料、獎盃也會一併被刪除。</user_delete_message>
+		<user_delete_msg>以下使用者將被刪除。</user_delete_msg>
+		<user_delete_message>若要刪除使用者，這位使用者的保存資料、獎盃也會一併被刪除。</user_delete_message>
```
Comment (Croden1999) [inline lang/zh-t.xml:542]: no need change, is based on the ps4 user management lang

File: lang/zh-t.xml [hunk 31]
```diff
@@ -538,11 +538,11 @@
...
 		<user_delete_warn>刪除使用者。
 確定要繼續嗎？</user_delete_warn>
-		<user_deleted>已刪除使用者。</user_deleted>
+		<user_deleted>使用者已刪除。</user_deleted>
```
Comment (Croden1999) [inline lang/zh-t.xml:545]: also here

File: lang/zh-t.xml [hunk 32]
```diff
@@ -552,35 +552,35 @@
...
 		<next>繼續</next>
-		<update_vita3k>確定要更新Vita3K吗？</update_vita3k>
+		<update_vita3k>要更新Vita3K嗎？</update_vita3k>
 		<later_version_already_installed>已安裝更新版本的Vita3K。</later_version_already_installed>
 		<latest_version_already_installed>已安裝最新版本的Vita3K。</latest_version_already_installed>
-		<new_features>{} 版本的追加機能</new_features>
+		<new_features>{}版本的追加機能</new_features>
 		<update>更新</update>
-		<version>{} 版本</version>
+		<version>{}版本</version>
```
Comment (Croden1999) [inline lang/zh-t.xml:566]: revert it

File: lang/zh-t.xml [hunk 33]
```diff
@@ -552,35 +552,35 @@
...
-下載完成后，Vita3K會自動重新啓動，並安裝新的版本。</downloading>
+下載完成后，Vita3K會自動重新啟動，並安裝新的版本。</downloading>
 		<not_complete_update>無法完成更新。</not_complete_update>
 		<next>繼續</next>
-		<update_vita3k>確定要更新Vita3K吗？</update_vita3k>
+		<update_vita3k>要更新Vita3K嗎？</update_vita3k>
 		<later_version_already_installed>已安裝更新版本的Vita3K。</later_version_already_installed>
 		<latest_version_already_installed>已安裝最新版本的Vita3K。</latest_version_already_installed>
-		<new_features>{} 版本的追加機能</new_features>
+		<new_features>{}版本的追加機能</new_features>
```
Comment (Croden1999) [inline lang/zh-t.xml:564]: also

File: lang/zh-t.xml [hunk 34]
```diff
@@ -552,35 +552,35 @@
 	</user_management>
 
 	<vita3k_update name="Vita3K更新">
- 		<new_version_available>發現新版的Vita3K。</new_version_available>
+ 		<new_version_available>Vita3K有可用的新版本。</new_version_available>
```
Comment (Croden1999) [inline lang/zh-t.xml:555]: is based on psvita lang

File: lang/zh-t.xml [hunk 35]
```diff
@@ -552,35 +552,35 @@
...
 		<back>返回</back>
 		<downloading>下載中…
-下載完成后，Vita3K會自動重新啓動，並安裝新的版本。</downloading>
+下載完成后，Vita3K會自動重新啟動，並安裝新的版本。</downloading>
 		<not_complete_update>無法完成更新。</not_complete_update>
 		<next>繼續</next>
-		<update_vita3k>確定要更新Vita3K吗？</update_vita3k>
+		<update_vita3k>要更新Vita3K嗎？</update_vita3k>
```
Comment (Croden1999) [inline lang/zh-t.xml:561]: also here

**Review by Croden1999 — CHANGES_REQUESTED** [review]

File: lang/zh-t.xml [hunk 36]
```diff
@@ -356,94 +356,94 @@
...
-			<download_firmware_font_package>下載字庫固件包</download_firmware_font_package>
-			<firmware_font_package_note>Firmware font package is needed for some applications and also for Asian regional font support. (Generally Recommended)</firmware_font_package_note>
-			<delete_firmware>Delete the firmware installation file?</delete_firmware>
+			<firmware_installing>安裝進行中，請稍候…</firmware_installing>
+			<successed_install_firmware>韌體已成功安裝。</successed_install_firmware>
+			<firmware_version>韌體版本：</firmware_version>
+			<no_font_exist>韌體字型套件不存在。
+請下載並安裝韌體字型套件。</no_font_exist>
+			<download_firmware_font_package>下載韌體字型套件</download_firmware_font_package>
+			<firmware_font_package_note>某些應用程式和亞洲區域字型支援需要韌體字型套件（一般建議安裝）。</firmware_font_package_note>
```
Comment (Croden1999) [inline lang/zh-t.xml:377]: no need change, use english, if no install firmware, here not display the normal text.

File: lang/zh-t.xml [hunk 37]
```diff
@@ -356,94 +356,94 @@
...
 		<firmware_install>
-			<firmware_installing>Installation in progress, please wait...</firmware_installing>
-			<successed_install_firmware>固件已成功安裝。</successed_install_firmware>
-			<firmware_version>固件版本：</firmware_version>
-			<no_font_exist>固件字庫包不存在。
-請下載安裝固件字庫包。</no_font_exist>
-			<download_firmware_font_package>下載字庫固件包</download_firmware_font_package>
-			<firmware_font_package_note>Firmware font package is needed for some applications and also for Asian regional font support. (Generally Recommended)</firmware_font_package_note>
-			<delete_firmware>Delete the firmware installation file?</delete_firmware>
+			<firmware_installing>安裝進行中，請稍候…</firmware_installing>
```
Comment (Croden1999) [inline lang/zh-t.xml:371]: also

File: lang/zh-t.xml [hunk 38]
```diff
@@ -356,94 +356,94 @@
...
+			<successed_install_firmware>韌體已成功安裝。</successed_install_firmware>
+			<firmware_version>韌體版本：</firmware_version>
+			<no_font_exist>韌體字型套件不存在。
+請下載並安裝韌體字型套件。</no_font_exist>
+			<download_firmware_font_package>下載韌體字型套件</download_firmware_font_package>
+			<firmware_font_package_note>某些應用程式和亞洲區域字型支援需要韌體字型套件（一般建議安裝）。</firmware_font_package_note>
+			<delete_firmware>刪除韌體安裝檔案？</delete_firmware>
```
Comment (Croden1999) [inline lang/zh-t.xml:378]: here

**Review by Croden1999 — CHANGES_REQUESTED** [review]

File: lang/zh-t.xml [hunk 39]
```diff
@@ -515,8 +515,8 @@
 	<trophy_collection>
 		<search>搜尋</search>
 		<delete_trophy>刪除獎盃</delete_trophy>
-		<trophy_deleted>刪除該使用者已保存的獎杯資訊。</trophy_deleted>
-		<details>詳細內容</details>
+		<trophy_deleted>這位使用者已保存的獎杯資訊將被刪除。</trophy_deleted>
+		<details>詳細資料</details>
```
Comment (Croden1999) [inline lang/zh-t.xml:519]: revert it.

**Review by Croden1999 — CHANGES_REQUESTED** [review]

File: lang/zh-t.xml [hunk 40]
```diff
@@ -356,94 +356,94 @@
...
-			<no_font_exist>固件字庫包不存在。
-請下載安裝固件字庫包。</no_font_exist>
-			<download_firmware_font_package>下載字庫固件包</download_firmware_font_package>
-			<firmware_font_package_note>Firmware font package is needed for some applications and also for Asian regional font support. (Generally Recommended)</firmware_font_package_note>
+			<successed_install_firmware>韌體已成功安裝。</successed_install_firmware>
+			<firmware_version>韌體版本：</firmware_version>
+			<no_font_exist>韌體字型套件不存在。
+請下載並安裝韌體字型套件。</no_font_exist>
+			<download_firmware_font_package>下載韌體字型套件</download_firmware_font_package>
+			<firmware_font_package_noteFirmware font package is needed for some applications and also for Asian regional font support. (Generally Recommended)</firmware_font_package_note>
```
Comment (Croden1999) [inline lang/zh-t.xml]: wrong format

**Review by Croden1999 — CHANGES_REQUESTED** [review]

File: lang/zh-t.xml (hunk 40 above)
Comment (Kaitul) [reply lang/zh-t.xml]: fixed

File: lang/zh-t.xml [hunk 41]
```diff
@@ -356,94 +356,94 @@
...
-			<firmware_not_detected>Firmware not detected. Installation is highly recommended</firmware_not_detected>
-			<firmware_font_not_detected>Firmware font not detected. Installing it is recommended for font text in Live Area</firmware_font_not_detected>
-			<live_area_help>Live Area幫助</live_area_help>
-			<browse_app>瀏覽應用程式列表</browse_app>
-			<browse_app_control>方向鍵、左操作桿、滾輪上/下或使用滑塊</browse_app_control>
-			<start_app>開啓應用</start_app>
-			<start_app_control>滾輪上/下或使用滑塊</start_app_control>
-			<show_hide>應用程式運行時顯示/隱藏</show_hide>
+			<control_setting>在控制設定中使用鍵盤組態集</control_setting>
+			<firmware_not_detected>未偵測到韌體，強烈建議安裝</firmware_not_detected>
```
Comment (Croden1999) [inline lang/zh-t.xml:420]: revert it, here use english

File: lang/zh-t.xml [hunk 42]
```diff
@@ -356,94 +356,94 @@
...
+			<firmware_font_not_detected>未偵測到韌體字型，強烈建議為Live Area字型文字安裝</firmware_font_not_detected>
```
Comment (Croden1999) [inline lang/zh-t.xml:421]: also

**Review by Croden1999 — CHANGES_REQUESTED** [review]

**Review by Croden1999 — DISMISSED** [review]

**Comment by Croden1999** [comment]
i need check it again

**Comment by Croden1999** [comment]
not good
image
image

File: lang/zh-t.xml [hunk 43]
```diff
@@ -352,98 +352,98 @@
...
-		<installed>已安裝：</installed>
-		<download_firmware>下載固件</download_firmware>
-		<download_font_package>下載字庫包</download_font_package>
-		<install_firmware_file>安裝固件文件</install_firmware_file>
-		<completed>已完成。</completed>
+		<installed>Installed:</installed>
+		<download_firmware>Download Firmware</download_firmware>
+		<download_font_package>Download Font Package</download_font_package>
+		<install_firmware_file>Install Firmware File</install_firmware_file>
+		<completed>Completed.</completed>
```
Comment (Croden1999) [inline lang/zh-t.xml:365]: the "completed" is normal, revert it.

File: lang/zh-t.xml [hunk 44]
```diff
@@ -352,98 +352,98 @@
...
 		<select_language>Select a language.</select_language>
-		<install_firmware>安裝固件。</install_firmware>
+		<install_firmware>Install Firmware.</install_firmware>
 		<install_highly_recommended>Installing both firmware files is highly recommended.</install_highly_recommended>
-		<installed>已安裝：</installed>
-		<download_firmware>下載固件</download_firmware>
-		<download_font_package>下載字庫包</download_font_package>
-		<install_firmware_file>安裝固件文件</install_firmware_file>
-		<completed>已完成。</completed>
+		<installed>Installed:</installed>
```
Comment (Croden1999) [inline lang/zh-t.xml:361]: also the "installed" is normal, revert it.

**Review by Croden1999 — CHANGES_REQUESTED** [review]

**Comment by Croden1999** [comment]
ur pr has conflicts that must be resolved, look this pr https://github.com/Vita3K/Vita3K/pull/2357

**Comment by Croden1999** [comment]
please Mr.@Zangetsu38 review this pr.

**Comment by Zangetsu38** [comment]
conflict, need rebase

File: lang/zh-t.xml [hunk 45]
```diff
@@ -352,98 +352,98 @@
 	</indicator>
 
 	<initial_setup>
-		<back>返回</back>
+		<back>Back</back>
```
Comment (Croden1999) [inline lang/zh-t.xml:355]: here is normal, revert it.

**Review by Croden1999 — CHANGES_REQUESTED** [review]

File: lang/zh-t.xml [hunk 46]
```diff
@@ -566,21 +566,21 @@
...
+	<welcome name="歡迎使用Vita3K">
 		<line_first>Vita3K PlayStation Vita模擬器</line_first>
 		<line_second>Vita3K是使用C++編寫的開源PlayStation Vita模擬器，適用於Windows、Linux、macOS以及Android。</line_second>
-		<line_third>目前模擬器仍處於早期階段，因此非常感謝任何反饋和測試。</line_third>
-		<line_fourth>要開始使用，請安裝PS Vita固件和字庫包。</line_fourth>
-		<download_firmware>下載固件</download_firmware>
-		<line_sixth_part_one>有關如何設置Vita3K的綜合指南可在</line_sixth_part_one>
+		<line_third>目前模擬器仍處於早期階段，因此非常感謝任何回饋和測試。</line_third>
+		<line_fourth>要開始使用，請安裝PS Vita韌體和字型套件。</line_fourth>
+		<download_firmware>下載韌體</download_firmware>
```
Comment (Croden1999) [inline lang/zh-t.xml:574]: the "download_firmware" use english

**Review by Croden1999 — CHANGES_REQUESTED** [review]

File: lang/zh-t.xml [hunk 47]
```diff
@@ -2,64 +2,64 @@
...
+		<create>建立</create>
 		<edit>編輯</edit>
 		<remove>移除</remove>
-		<open_folder>打开資料夾</open_folder>
-		<addcont>追加資料</addcont>
-		<license>許可證</license>
-		<shader_cache>着色器緩存</shader_cache>
-		<shader_log>着色器日志</shader_log>
+		<open_folder>開啟資料夾</open_folder>
+		<addcont>追加內容</addcont>
```
Comment (Croden1999) [inline lang/zh-t.xml:52]: is based on psvita devkit/testkit lang

**Review by Croden1999 — CHANGES_REQUESTED** [review]

**Comment by Croden1999** [comment]
ur pr has conflicts that must be resolved

**Review by Croden1999 — APPROVED** [review]

**Comment by Croden1999** [comment]
please Mr.@Zangetsu38 review it

**Review by Zangetsu38 — APPROVED** [review]

**Pull request merged**